week3/homework/src/index.js

```diff
@@ -1,3 +1,130 @@
 'use strict';
 
-// Write the homework code in this file
+
+const express = require('express');
+const uuidv4 = require('uuid/v4');
+
+const app = express();
+const PORT = 3000;
+
+
+const {
+    writeFile,
+    readFile
+} = require('./todos.js');
+
+
+
+app.use(express.json());
+
+
+app.get('/todos', (req, res) => {
+    readFile().then(data => {
+        res.json(data);
+        res.end();
+    });
+
+});
+
+app.post('/todos', (req, res) => {
+    readFile()
+        .then(todos => {
+            const todoToAdd = req.body.todo;
+            todoToAdd.id = uuidv4();
+            todos.push(todoToAdd);
+            writeFile(todos)
+                .then(() => {
+                    res.send(`Wrote ${req.body.todo}`);
+                    res.end();
+                });
+        });
+
+});
+
+
+app.delete('/todos/:id', (req, res) => {
+    const removeId = req.params.id;
+    readFile()
+        .then(todos => {
+            const newTodos = todos.filter(todo => todo.id !== removeId);
+            writeFile(newTodos)
+                .then(() => {
+                    res.send(`Removed ${removeId}`);
+                    res.end();
+                });
+        });
+
+});
+app.delete('/todos', (req, res) => {
+    readFile()
```

Comment:
You are reading the file and not doing anything with the result before overwriting it.